lib/bonny/crd.ex

```diff
@@ -100,16 +99,21 @@ defmodule Bonny.CRD do
     version: v1
   ```
   """
-  @spec to_manifest(Bonny.CRD.t()) :: map
-  def to_manifest(%CRD{} = crd) do
+  @spec to_manifest(Bonny.CRD.t(), String.t()) :: map
+  def to_manifest(%CRD{} = crd, api_version) do
+    spec = case api_version do
+      "apiextensions.k8s.io/v1" -> format_spec_v1(crd)
+      _ -> format_spec(crd)
```

Comment:
Would you mind renaming this old private `format_spec/1` method to `format_spec_v1beta1` for clarity?

Reply:
makes sense